sound/soc/sof/ipc4-topology.c

```diff
@@ -1251,7 +1251,7 @@ sof_ipc4_prepare_copier_module(struct snd_sof_widget *swidget,
 		ipc4_copier = (struct sof_ipc4_copier *)dai->private;
 		copier_data = &ipc4_copier->data;
 		available_fmt = &ipc4_copier->available_fmt;
-		if (dir == SNDRV_PCM_STREAM_CAPTURE) {
+		if (dir == SNDRV_PCM_STREAM_CAPTURE || ipc4_copier->dai_type == SOF_DAI_INTEL_SSP) {
```

Comment:
Commit message: can we use the same term we are using for the components?
`ASoC: SOF: ipc4-topology: Use 32bit sample size for copier on Intel SSP`

s/FW/The firmware

Reply:
You should update the comment down at line 1259 as well.

Reply:
@RanderWang the change looks good but do you have an issue that this PR fixes?

Reply:
> @RanderWang the change looks good but do you have an issue that this PR fixes?

sorry missed the issue in the descriptoin

Reply:
@ranj063 yong changed nocodec tplg but made no effect https://github.com/thesofproject/sof/issues/6595#issuecomment-1317386461.